src/spaces/compartment.jl

```diff
@@ -144,7 +146,7 @@ function nearby_ids(pos::ValidPos, model::ABM{<:AbstractAgent,<:CompartmentSpace
             max_distance = r
         end
         final_ids = Int[]
-        allcells = grid_space_neighborhood(CartesianIndex(focal_cell), model, newr)
+        allcells = grid_space_neighborhood(CartesianIndex(focal_cell), model, newr+1)
```

Comment:
Notice that the extra radius is not 1 but `sqrt(D)` with `D` the dimensionality of the space